### PR TITLE
{common-lisp}Issue #1157 comment for '-p'

### PR DESCRIPTION
**Pull request by thePsguy**
Comment explaining '-p', in line 178. issue #1157

### CODE REVIEW

File: common-lisp.html.markdown [hunk 1]
```diff
@@ -175,7 +175,8 @@ nil                  ; for false - and the empty list
               :age 5))
 *rover* ; => #S(DOG :NAME "rover" :BREED "collie" :AGE 5)
 
-(dog-p *rover*) ; => t  ;; ewww)
+(dog-p *rover*) ; => true  #| -p signifies "predicate". It's used to
+                              check if *rover* is an instance of dog.|#
```
Comment (vendethiel) [inline common-lisp.html.markdown]: should the end not be spaced?

Comment (thePsguy) [inline common-lisp.html.markdown]: my bad. fixed.

**Comment by vendethiel** [comment]
Thanks,  sorry for nitpicking :P.

**Comment by vendethiel** [comment]
:heart_eyes: :heart_eyes_cat:

**Comment by thePsguy** [comment]
This is what open source is for! I should Thank You for the nitpicking! :D

**Pull request merged**